agent/Shooter.py

```diff
@@ -85,23 +104,29 @@ def select_card(self,
         else:
             # We are following a trick
             if self.is_early(partial_state):
-                # TODO is_early most likely counts too much time when it comes to whether
-                # we should follow with low cards or not so we can update it later
                 lowest = self.get_lowest(self.cards_in_hand)
                 return random.choice(lowest)
             else:
                 # Later in the game so try to take tricks from players
-                # TODO we should be checking if we can even follow suit to know
                 # if we are capable of taking the trick
                 if (self.is_last):
```

Comment:
`if (self.is_last):` ends up always being true. I believe this to be the case after debugging and checking if the player is last. I changed it to `if(self.is_last(partial_state)):`.

Reply:
It must have been creating a variable because I forgot to make it a function call. Fixed.